fiscalsim_us/parameters/gov/states/mn/tax/income/amt/std_deduct_phase_out.yaml

```diff
@@ -0,0 +1,21 @@
+description: MN alternative minimum tax standard deduction
+metadata:
```

Comment:
@mitchellpound. As in my first comment in the `standard_deduct.yaml` parameter. This parameter is misnamed and should be the `standard_deduct.yaml` parameter. Furthermore, this parameter has only a subset of the information in the current `fractional_income_threshold.yaml` parameter, which is no longer used in this PR. You might consider just using the `fractional_income_threshold.yaml` parameter. Another approach is to incorporate the extra information in that parameter into this parameter.